### DISCUSSION TITLE
Reusable Workflow as required check by a protected branch

### DISCUSSION THREAD

**Discussion by noamcohen97** [community/community#35934] [Actions]
I have a reusable workflow, which I use during build on pull requests.
The workflow is sometime skipped, but I still want it to be required before merging to the main branch.
It seems like after run, the workflow name is changing, and it is not recognized as the workflow specified in the required checks section in the protected branch settings.
If I use the name it gets after the rename occurs, it's not recognized when it skipped and merge is blocked.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @noamcohen97 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
